### PR TITLE
[ci/windows] remove obsolete 'tools' in the name

### PR DESCRIPTION
**Pull request by umarcor**

### CODE REVIEW

File: .github/workflows/windows_builds.yml [hunk 1]
```diff
@@ -23,15 +23,15 @@ jobs:
...
+          - name: Editor (tests=yes)
             cache-name: windows-editor
             target: editor
             tests: true
             # Skip debug symbols, they're way too big with MSVC.
             sconsflags: debug_symbols=no vsproj=yes
             bin: "./bin/godot.windows.editor.x86_64.exe"
 
-          - name: Template (target=template_release, tools=no)
+          - name: Template (tools=no)
```
Comment (akien-mga) [inline .github/workflows/windows_builds.yml:34]: `tools=no` is obsolete, should be removed.

**Comment by akien-mga** [comment]
Templates can be either `target=template_release` or `target=template_debug`. We don't have any of the latter currently but we may at some point in the future, so being explicit was useful.

But I don't feel too strongly either way, this doesn't seem particularly needed nor useful, but I don't mind the change if there's support for it.

**Comment by umarcor** [comment]
@akien-mga changed to just remove the obsolete 'tools'.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.0.3.

**Pull request merged**